README.md

```diff
@@ -10,14 +10,14 @@
 We offer a suite of prompt-decoders, albeit with focus on open source LLMs compatible with [FastChat](https://github.com/lm-sys/FastChat?tab=readme-ov-file#supported-models) (e.g., Vicuna, Zephyr, etc.). Some of the code in this repository is borrowed from [RankGPT](https://github.com/sunnweiwei/RankGPT)!
 
 # Releases
-current_version = 0.2.8
+current_version = 0.12.8
 
 ## 📟 Instructions
 
 ### Create Conda Environment
 
 ```bash
-conda create -n rankllm python=3.10
+conda create -n rankllm python=3.12
```

Comment:
Are we dependent on 3.12? Because I'm not sure if most people have it yet.

Reply:
tested with 3.10, works, reverted